ci/release/update-version.sh

```diff
@@ -106,3 +106,7 @@ sed_runner "s/VERSION=\".*\"/VERSION=\"${NEXT_FULL_JAVA_TAG}\"/g" java/build.sh
 for FILE in java/*/pom.xml; do
   sed_runner "/<!--CUVS_JAVA#VERSION_UPDATE_MARKER_START-->.*<!--CUVS_JAVA#VERSION_UPDATE_MARKER_END-->/s//<!--CUVS_JAVA#VERSION_UPDATE_MARKER_START--><version>${NEXT_FULL_JAVA_TAG}<\/version><!--CUVS_JAVA#VERSION_UPDATE_MARKER_END-->/g" "${FILE}"
 done
+
+sed_runner "s| ${CURRENT_SHORT_TAG} | ${NEXT_SHORT_TAG} |g" java/README.md
+sed_runner "s|-${CURRENT_SHORT_TAG}\.[0-9]\.jar|-${NEXT_FULL_JAVA_TAG}\.jar|g" java/examples/README.md
+sed_runner "s|\/${CURRENT_SHORT_TAG}\.[0-9]\/|/${NEXT_FULL_JAVA_TAG}/|g" java/examples/README.md
```

Comment:
Using the current version can be fragile. And `[0-9]` has varying support per platform.

Tested these and seems to work:
```suggestion
sed_runner "s| CuVS [[:digit:]]\{2\}\.[[:digit:]]\{2\} | CuVS ${NEXT_SHORT_TAG} |g" java/README.md
sed_runner "s|-[[:digit:]]\{2\}\.[[:digit:]]\{2\}\.[[:digit:]]\{1,2\}\.jar|-${NEXT_FULL_JAVA_TAG}\.jar|g" java/examples/README.md
sed_runner "s|/[[:digit:]]\{2\}\.[[:digit:]]\{2\}\.[[:digit:]]\{1,2\}/|/${NEXT_FULL_JAVA_TAG}/|g" java/examples/README.md
```

Reply:
fixed